hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/LinuxContainerExecutor.java

```diff
@@ -525,6 +580,23 @@ public int launchContainer(ContainerStartContext ctx) throws IOException {
         logOutput(diagnostics);
         container.handle(new ContainerDiagnosticsUpdateEvent(containerId,
             diagnostics));
+        if (exitCode == LinuxContainerExecutorExitCode.
```

Comment:
Right.  Forgot about that.  You'd basically have to recreate the same code in the enum to get an instance from an int.

Maybe add an equals() method to the enum that can compare against ints as well?  Maybe not worth it.  Just shortening the enum name may be enough...